pkg/controller/plan/migration.go

```diff
@@ -1031,6 +1031,15 @@ func (r *Migration) execute(vm *plan.VMStatus) (err error) {
 				Message:  "The VM migration has SUCCEEDED.",
 				Durable:  true,
 			})
+		// Detach LUNs
```

Comment:
right and do we want to set the SUCCEEDED condition when we fail to detach the disk in ovirt?

Reply:
yes? i'm looking on this as "best effort"

Reply:
but for users it may look differently if we risk corrupting the disk :)
I think we should make it obvious to the user when we fail to detach and ideally to keep monitoring the state of the disk, so if the user detaches the disk on oVirt - we can proceed

Reply:
@liranr23 I don't see this happening often that we fail to detach a disk at the end of the migration but when it happens it may have severe implications, how about failing the migration in this case for now and later try to monitor it?

Reply:
ok